### DISCUSSION TITLE
Phishing alert

### DISCUSSION THREAD

**Discussion by Luuan11** [community/community#40103] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hello team,

Could someone help me? 
I have a phising alert on my github, on my instagram clone

repo: https://github.com/Luuan11/clone-instagram-login

Deploy: https://luuan11.github.io/clone-instagram-login/

https://www.virustotal.com/gui/domain/luuan11.github.io

![image](https://user-images.githubusercontent.com/79935555/204051412-6931844d-8a7b-4034-b5a3-917cdd98f6f7.png)

if someone know, what to do, tell me please

**Comment by airtower-luna** [comment, marked as answer]
"Instagram clone" is the key here: Someone noticed you're imitating the login form of another website, so you ended up on block lists for phishing. The exact process varies depending on the block list, but generally you'll have to take down the clone and then request a review.

The most common block list is Google's Safe Browsing one, so you might want to start with [their documentation](https://developers.google.com/search/docs/monitor-debug/security/social-engineering).

**Reply by Luuan11** [reply]
So, if a remove my repo, probably will fix ?

**Reply by Luuan11** [reply]
check this https://www.virustotal.com/gui/domain/luuan11.github.io, now all my github.io have this alert

**Reply by airtower-luna** [reply]
> So, if a remove my repo, probably will fix ?

No, it's only the first step. You'll then have to request a review from the block list.

> now all my github.io have this alert

That's expected, the block lists usually mark whole domains.

**Reply by Luuan11** [reply]
Ok, I'll do this, and do you have a link to request this review, ?

But, for now, ty for you help, after all this thing, I'll not deploy the instagram clone anymore, ty ty

**Reply by airtower-luna** [reply]
Here, it's in a section of the documentation I mentioned above: https://developers.google.com/search/docs/monitor-debug/security/social-engineering#fixing-the-problem

There seem to be two ways now (in steps 2 and 4), I'm not sure which one's preferable. :sweat_smile:

**Reply by Luuan11** [reply]
I still have the problem ;-;
